rootfs/api/views.py

```diff
@@ -86,11 +87,14 @@ def destroy(self, request, **kwargs):
 
         # A user can not be removed without apps changing ownership first
         if len(models.App.objects.filter(owner=target_obj)) > 0:
-            msg = '{} still has applications assigned. Delete or transfer ownership'.format(str(target_obj))  # noqa
+            msg = '{} still has applications assigned. Please delete or transfer ownership'.format(str(target_obj))  # noqa
```

Comment:
Okay if it's too much trouble I'll just remove this commit for now. It was more of a QoL change. I'll make a separate PR for this if I feel motivated :)